### PR TITLE
BZ#1700770 Edit service creation command

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1700770

### CODE REVIEW

**Comment by codyhoag** [comment]
@sunilcio PTAL?

**Comment by sunilcio** [comment]
@codyhoag I see we are creating a service object with custom port. We need to change step 4 to `oc create -f mysql.yaml` to create service object from the yaml.

**Comment by codyhoag** [comment]
@sunilcio can you take a second look with my updates? Thanks!

**Comment by sunilcio** [comment]
@codyhoag thanks, LGTM

**Comment by bmcelvee** [comment]
LGTM!

File: dev_guide/expose_service/expose_internal_ip_nodeport.adoc [hunk 1]
```diff
@@ -78,16 +78,16 @@ spec:
     name: mysql
 ----
 
-. Execute the following command to xref:../../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[create the service]:
+. Execute the following command to create the service:
```
Comment (rolfedh) [inline dev_guide/expose_service/expose_internal_ip_nodeport.adoc:81]: To paraphrase the IBM Style Guide: Use **execute** only if a simpler, less technical term, such as "run" or a more specific term, such as "run" or "start" is inappropriate.

Comment (codyhoag) [reply dev_guide/expose_service/expose_internal_ip_nodeport.adoc:81]: Thanks! Submitted #20901 to update this.

**Pull request merged**